_attributes/common-attributes.adoc

```diff
@@ -289,3 +289,33 @@ endif::[]
 :rh-dev-hub: Red Hat Developer Hub
 //IBU
 :lcao: Lifecycle Agent
+
+
+// Cluster API Providers
+// Cluster API Provider Amazon Web Services (AWS)
+:capa-first: Cluster API Provider Amazon Web Services (AWS)
```

Comment:
There's nothing technically wrong with these attributes, but I'd like to suggest renaming them slightly. The names would be a bit longer, but I believe they would be easier to remember and understand at a glance if we separate the "Cluster API Provider" part as `cap`, then use common acronyms for the providers as used elsewhere in the attributes.

For example, I would change the name of this attribute to `:cap-aws-first:`.

Feel free to disregard this entirely if you disagree, of course. There's nothing wrong with them as-is, I just like reducing mental juggling trying to remember where to use, say, `gcp` and where it's just `g`. :slightly_smiling_face: